webpack-test/configCases/source-map/srouce-map-loader/index.js

```diff
@@ -0,0 +1,8 @@
+it("should include a.ts in SourceMap", function() {
```

Comment:
Additional test cases should be added to `packages/rspack/tests`. Would you please move?

Reply:
Folder moved.